### PR TITLE
workflows: antihep keyword list

### PR DESCRIPTION
**Pull request by jalavik**

### CODE REVIEW

File: inspirehep/kbs/corepar.kb [hunk 1]
```diff
@@ -0,0 +1,798 @@
...
+Z0(1780)---Z0(1780)
+Z0(1865)---Z0(1865)
+Z1(1725)---Z1(1725)
+Z1(1900)---Z1(1900)
+Z1(2150)---Z1(2150)
+Z1(2500)---Z1(2500)
+Z2(4250)---Z2(4250)
+zinc---zinc
+Zino---Zino
+zirconium---zirconium
```
Comment (kaplun) [inline inspirehep/kbs/corepar.kb]: Can you add a `README.rst` file to explain the meaning of these files?

**Comment by kaplun** [comment]
What is antihep?

**Comment by jalavik** [comment]
@kaplun This is a list of core HEP keywords that should be filtered out of the core keywords list. @fschwenn can explain better, but basically these CORE keywords should not count towards the metric number of CORE keyword, nor be displayed.

**Comment by kaplun** [comment]
I love how the coverage keeps on increasing and increasing... :smile:

**Comment by fschwenn** [comment]
Jan's explanation is completely right. Just the name of the issue is a bit misleading as corepar.kb should not be mixed up with https://github.com/inspirehep/inspire-next/blob/production/inspirehep/base/classifierext/taxonomies/antiHEPont.rdf which is a real 'anti'-list containing keywords like 'biology' which indicate that the content is not related to HEP.

**Pull request merged**